Rakefile

```diff
@@ -11,7 +11,8 @@ namespace :assets do
   desc 'Update Foundation for Sites assets'
   task update: :clean do
     sh 'bower install'
-    sh 'cp -R bower_components/foundation-sites/js/* vendor/assets/js/'
+    sh 'cp -R bower_components/foundation-sites/dist/js/plugins/* vendor/assets/js/'
```

Comment:
They are required because then they are picked up by the rails asset pipeline.
So technically the `foundation-sites/dist/js/foundation.js` would be the same as the `foundation.js.es6` file this generates.
The main difference would be that in development rails will add hashes to the file names and not combine them but include them in the HTML this can be easier to debug sometime.